### DISCUSSION TITLE
I wish that Sumatra would treat "remember recent read files and its progress" and "open them next time start Sumatra" as different function calls.

### DISCUSSION THREAD

**Discussion by vacarogit** [sumatrapdfreader/sumatrapdf#2796] [Ideas]
Dear respected Author,

SumatraPDF is a great reader I wish that Sumatra would treat "remember recent read files and its progress" and "open them next time start Sumatra" as different function calls.

So far these two options are binded, you know what I mean? I read many files at same time, but I don't need to open all of them open to suck up ram resources and time, but I love the last read progress can be kept also.

Thank you!!

**Comment by GitHubRulesOK** [comment]
to use a History you need to `Remember opened files`
to remember last page you need `Remember settings for each (all) documents`
To open last session you need to close session without closing tabs e.g Ctrl+Q (File exit) and `RestoreSession = true`
**To open new session without old tabs set `RestoreSession = false`**
To not get HomeTab set `NoHomeTab = true`

There are also new commands for **close other tabs** or **close tabs to the right** before closing session so you get less on restart

**Comment by kjk** [comment]
To add to @GitHubRulesOK response: those are advanced settings. See https://www.sumatrapdfreader.org/docs/Advanced-options-settings and https://www.sumatrapdfreader.org/settings/settings

**Comment by daSinzi** [comment]
I think a new tab button is the solution. On the new tab (here the empty tab) you can see all recent and favorites. In empty tab you can only open one file.
